linux_os/guide/system/accounts/accounts-pam/display_login_attempts/tests/correct_value.pass.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/bash
-# platform = Red Hat Enterprise Linux 7,multi_platform_ol,multi_platform_ubuntu
+# platform = multi_platform_all
 
 {{% if product in ["sle12", "sle15"] or 'ubuntu' in product %}}
-rm -f /etc/pam.d/login
-echo "session required pam_lastlog.so showfailed" >> /etc/pam.d/login
+{{% set pam_lastlog_path = "/etc/pam.d/login" %}}
 {{% else %}}
-rm -f /etc/pam.d/postlogin
-echo "session required pam_lastlog.so showfailed" >> /etc/pam.d/postlogin
+{{% set pam_lastlog_path = "/etc/pam.d/postlogin" %}}
 {{% endif %}}
+
+rm -f {{{ pam_lastlog_path }}}
+
+cat <<EOF > {{{ pam_lastlog_path }}}
+session     optional                   pam_umask.so silent
+session     [success=1 default=ignore] pam_succeed_if.so service !~ gdm* service !~ su* quiet
+session     [default=1]                pam_lastlog.so nowtmp showfailed
```

Comment:
perhaps make control part of the if above and refer to it here?

Reply:
Instead of creating more conditionals, I ensured two `pass` scripts with different controls. It should be enough for testing purposes. https://github.com/ComplianceAsCode/content/pull/10979/commits/21ae1e154f60c185a96daf401782665115e15242